pkgpanda/build/test_util.py

```diff
@@ -0,0 +1,86 @@
+import pkgpanda.build
+
+
+def test_hash_files_in_folder(tmpdir):
+    hash_files_in_folder = pkgpanda.build.hash_files_in_folder
+
+    with tmpdir.as_cwd():
+        # Empty folder
```

Comment:
Feel free to merge - it's not something that I think it's important enough to discuss endlessly and die for if necessary ;)

I was following these guidelines in i.e. Datastore tests - feel free to have a look and see if this is more readable for you: https://github.com/mesosphere/bouncer/blob/master/tests/datastore/test_datastore.py#L278